springboottrades-quotes/src/main/resources/logback.xml

```diff
@@ -0,0 +1,57 @@
+<configuration>
+	<conversionRule conversionWord="clr" converterClass="org.springframework.boot.logging.logback.ColorConverter" />
+	<conversionRule conversionWord="wex" converterClass="org.springframework.boot.logging.logback.WhitespaceThrowableProxyConverter" />
+
+	<property name="CONSOLE_LOG_PATTERN" value="%clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} %clr(%5p) %clr(${PID:- }){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%replace(%xException){'\n','&#x2028;'}%nopex%n"/>
```

Comment:
@dpinto-pivotal I think that if we could upgrade to spring-boot 1.3.0.RC1 (see https://github.com/spring-projects/spring-boot/issues/3367) we could avoid having to specify the entire log back.xml and just override the `CONSOLE_LOG_PATTERN` in `application.yaml` to the `spring_cloud` profile

Is an upgrade to spring-boot 1.3.0.RC1 feasible?